receiver/nginxreceiver/scraper.go

```diff
@@ -50,7 +51,7 @@ func newNginxScraper(
 func (r *nginxScraper) start(_ context.Context, host component.Host) error {
 	httpClient, err := r.cfg.ToClient(host.GetExtensions())
 	if err != nil {
-		r.logger.Error("nginx", zap.Error(err), zap.String("status_code", "INVALID_ARGUMENT"))
+		r.logger.Error("nginx", zap.Error(err), zap.String("status_code", codes.InvalidArgument.String()))
```

Comment:
I believe we don't need to call `String()` explicitly, since `zap.String` probably handles this.